docs/sources/gard.md

```diff
@@ -1,17 +1,90 @@
 # MONDO - GARD Alignment
 
-**Source name:** GARD
+**Source name:** Genetic and Rare Diseases
 
-**Source description:** The GARD mapping is mainly curated through the GARD-ORDO and GARD-OMIM mappings. From the GARD developers, we have confirmed that, For a given GARD ID,  GARD guarantees no other concept will be assigned in the future.   The concept will not be assigned to a different GARD ID, but this GARD ID may be retired  when a similar but more accurate concept is created under a new GARD ID to replace this GARD ID.
+**Source description:**  Genetic and Rare Diseases (GARD) Information Center is a public health resource that aims to support people living with a rare disease and their families with free access to reliable, easy to understand information. 
 
+**Homepage:** https://rarediseases.info.nih.gov/
 
-**Homepage:** None
+## Comments about this source:
```

Comment:
## Recreated GARD docs
@matentzn Recently my docs got erased ;(. The git blame shows this was a commit from you on August 23rd.

I imagine that some pipeline being ran ended up overwriting the docs.

This is something I was wondering about before but never brought up. Since these `sources/SOURCE.md` files are dynamically generated, but there is a part in there that can be written statically/manually, this creates a problem.

Some possible solutions:
a. If the `sources/SOURCE.md` file already exists, don't overwrite it; do nothing.
b. Have a second `SOURCE-static.md` somewhere else (possibly outside of `docs/`), and when `sources/SOURCE.md` is dynamically created, it can check for `SOURCE-static.md` and interpolate that text inside of it if such a file exists.
c. Have a second `SOURCE-static.md`, but have `source/SOURCE.md` have a dynamically generated hyperlink to the static file if said static file exists.

Let me know if you want me to write up an issue about this.

Reply:
Oh no! 

Sorry about that! Would you see it as a possibility to curate your docs somehow in the associated yaml file? https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata/doid.yml?

Or is it too inflexible? This is so everything we say about one source can be kept in one place, rather than two. But I am open to add two files.

Reply:
It's alright. I think your solution is fine too. I was just trying now to run a test and see if it had any problems ideally generating markdown from within the metadata yml `description`. But I'm having trouble running it; can you help?

I ran `sh run.sh make update_repo` twice, but it didn't regenerate those markdown files. Is there another command that's supposed to do this?

<details><summary>Log</summary>
<p>

```
(mondo-ingest) sh run.sh make update_repo
Running  with -Xmx20G of memory for ROBOT and Java-based pipeline steps.
mondo-ingest.Makefile:39: warning: overriding recipe for target 'components/omim.owl'
Makefile:447: warning: ignoring old recipe for target 'components/omim.owl'
mondo-ingest.Makefile:54: warning: overriding recipe for target 'components/ordo.owl'
Makefile:458: warning: ignoring old recipe for target 'components/ordo.owl'
mondo-ingest.Makefile:73: warning: overriding recipe for target 'components/ncit.owl'
Makefile:436: warning: ignoring old recipe for target 'components/ncit.owl'
mondo-ingest.Makefile:84: warning: overriding recipe for target 'components/doid.owl'
Makefile:392: warning: ignoring old recipe for target 'components/doid.owl'
mondo-ingest.Makefile:106: warning: overriding recipe for target 'component-download-icd10cm.owl'
Makefile:410: warning: ignoring old recipe for target 'component-download-icd10cm.owl'
mondo-ingest.Makefile:112: warning: overriding recipe for target 'components/icd10cm.owl'
Makefile:414: warning: ignoring old recipe for target 'components/icd10cm.owl'
mondo-ingest.Makefile:127: warning: overriding recipe for target 'components/icd10who.owl'
Makefile:425: warning: ignoring old recipe for target 'components/icd10who.owl'
mondo-ingest.Makefile:142: warning: overriding recipe for target 'component-download-gard.owl'
Makefile:399: warning: ignoring old recipe for target 'component-download-gard.owl'
mondo-ingest.Makefile:146: warning: overriding recipe for target 'components/gard.owl'
Makefile:403: warning: ignoring old recipe for target 'components/gard.owl'
mondo-ingest.Makefile:151: warning: overriding recipe for target 'mondo-ingest-full.owl'
Makefile:588: warning: ignoring old recipe for target 'mondo-ingest-full.owl'
mondo-ingest.Makefile:545: warning: overriding recipe for target 'help'
Makefile:659: warning: ignoring old recipe for target 'help'
sh ../scripts/update_repo.sh
This (experimental) update script will create a new repo according to your config file. It will:
(1) overwrite your repositories Makefile, ODK sparql queries (your custom queries wont be touched) and docker wrapper (run.sh).
(2) and add missing files, if any.
INFO:root:  Copying: /tools/templates/_dynamic_files.jinja2 -> target/mondo-ingest/_dynamic_files.jinja2
INFO:root:  Copying: /tools/templates/README.md.jinja2 -> target/mondo-ingest/README.md.jinja2
INFO:root:  Copying: /tools/templates/CODE_OF_CONDUCT.md -> target/mondo-ingest/CODE_OF_CONDUCT.md
INFO:root:  Copying: /tools/templates/CONTRIBUTING.md.jinja2 -> target/mondo-ingest/CONTRIBUTING.md.jinja2
INFO:root:  Copying: /tools/templates/issue_template.md.jinja2 -> target/mondo-ingest/issue_template.md.jinja2
INFO:root:  Copying: /tools/templates/.gitignore.jinja2 -> target/mondo-ingest/.gitignore.jinja2
INFO:root:Applying templates
INFO:root:  Unpacking: target/mondo-ingest/_dynamic_files
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/mondo-ingest-edit.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/mondo-ingest-idranges.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/mondo-ingest.Makefile
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/ro_import.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/omo_import.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/ro_terms.txt
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/omo_terms.txt
INFO:root:  Unpacking into: target/mondo-ingest/src/sparql/mondo-ingest_terms.sparql
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/doid.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/gard.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/icd10cm.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/icd10who.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/ncit.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/omim.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/ordo.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/profile.txt
INFO:root:  Unpacking into: target/mondo-ingest/.github/workflows/qc.yml
INFO:root:  Unpacking into: target/mondo-ingest/docs/index.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/cite.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/contributing.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/index.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/ContinuousIntegration.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/EditorsWorkflow.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/ReleaseWorkflow.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/RepoManagement.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/RepositoryFileStructure.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/SettingUpDockerForODK.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/UpdateImports.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/components.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/ManageDocumentation.md
INFO:root:  Unpacking into: target/mondo-ingest/mkdocs.yaml
INFO:root:  Unpacking into: target/mondo-ingest/.github/workflows/docs.yml
INFO:root:  Compiling: /tools/templates/README.md.jinja2 -> target/mondo-ingest/README.md
INFO:root:  Compiling: /tools/templates/CONTRIBUTING.md.jinja2 -> target/mondo-ingest/CONTRIBUTING.md
INFO:root:  Compiling: /tools/templates/issue_template.md.jinja2 -> target/mondo-ingest/issue_template.md
INFO:root:  Compiling: /tools/templates/.gitignore.jinja2 -> target/mondo-ingest/.gitignore
INFO:root:Applying templates
INFO:root:  Copying: /tools/templates/src/scripts/validate_id_ranges.sc -> target/mondo-ingest/src/scripts/validate_id_ranges.sc
INFO:root:  Copying: /tools/templates/src/scripts/update_repo.sh.jinja2 -> target/mondo-ingest/src/scripts/update_repo.sh.jinja2
INFO:root:Applying templates
INFO:root:  Compiling: /tools/templates/src/scripts/update_repo.sh.jinja2 -> target/mondo-ingest/src/scripts/update_repo.sh
INFO:root:  Copying: /tools/templates/src/sparql/postprocess-module.ru -> target/mondo-ingest/src/sparql/postprocess-module.ru
INFO:root:  Copying: /tools/templates/src/sparql/preprocess-module.ru -> target/mondo-ingest/src/sparql/preprocess-module.ru
INFO:root:  Copying: /tools/templates/src/sparql/README.md.jinja2 -> target/mondo-ingest/src/sparql/README.md.jinja2
INFO:root:  Copying: /tools/templates/src/sparql/edges.sparql -> target/mondo-ingest/src/sparql/edges.sparql
INFO:root:  Copying: /tools/templates/src/sparql/basic-report.sparql -> target/mondo-ingest/src/sparql/basic-report.sparql
INFO:root:  Copying: /tools/templates/src/sparql/obsoletes.sparql -> target/mondo-ingest/src/sparql/obsoletes.sparql
INFO:root:  Copying: /tools/templates/src/sparql/synonyms.sparql -> target/mondo-ingest/src/sparql/synonyms.sparql
INFO:root:  Copying: /tools/templates/src/sparql/inject-subset-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-subset-declaration.ru.jinja2
INFO:root:  Copying: /tools/templates/src/sparql/labels.sparql -> target/mondo-ingest/src/sparql/labels.sparql
INFO:root:  Copying: /tools/templates/src/sparql/xrefs.sparql -> target/mondo-ingest/src/sparql/xrefs.sparql
INFO:root:  Copying: /tools/templates/src/sparql/inject-synonymtype-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-synonymtype-declaration.ru.jinja2
INFO:root:  Copying: /tools/templates/src/sparql/terms.sparql -> target/mondo-ingest/src/sparql/terms.sparql
INFO:root:  Copying: /tools/templates/src/sparql/subsets-labeled.sparql -> target/mondo-ingest/src/sparql/subsets-labeled.sparql
INFO:root:  Copying: /tools/templates/src/sparql/simple-seed.sparql -> target/mondo-ingest/src/sparql/simple-seed.sparql
INFO:root:Applying templates
INFO:root:  Compiling: /tools/templates/src/sparql/README.md.jinja2 -> target/mondo-ingest/src/sparql/README.md
INFO:root:  Compiling: /tools/templates/src/sparql/inject-subset-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-subset-declaration.ru
INFO:root:  Compiling: /tools/templates/src/sparql/inject-synonymtype-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-synonymtype-declaration.ru
INFO:root:  Copying: /tools/templates/src/ontology/run.bat.jinja2 -> target/mondo-ingest/src/ontology/run.bat.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/README-editors.md.jinja2 -> target/mondo-ingest/src/ontology/README-editors.md.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/Makefile.jinja2 -> target/mondo-ingest/src/ontology/Makefile.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/catalog-v001.xml.jinja2 -> target/mondo-ingest/src/ontology/catalog-v001.xml.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/run.sh.jinja2 -> target/mondo-ingest/src/ontology/run.sh.jinja2
INFO:root:Applying templates
INFO:root:  Compiling: /tools/templates/src/ontology/run.bat.jinja2 -> target/mondo-ingest/src/ontology/run.bat
INFO:root:  Compiling: /tools/templates/src/ontology/README-editors.md.jinja2 -> target/mondo-ingest/src/ontology/README-editors.md
INFO:root:  Compiling: /tools/templates/src/ontology/Makefile.jinja2 -> target/mondo-ingest/src/ontology/Makefile
INFO:root:  Compiling: /tools/templates/src/ontology/catalog-v001.xml.jinja2 -> target/mondo-ingest/src/ontology/catalog-v001.xml
INFO:root:  Compiling: /tools/templates/src/ontology/run.sh.jinja2 -> target/mondo-ingest/src/ontology/run.sh
bINFO:root:Created files:
INFO:root:  File: target/mondo-ingest/CODE_OF_CONDUCT.md
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest-edit.owl
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest-idranges.owl
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest.Makefile
INFO:root:  File: target/mondo-ingest/src/ontology/imports/ro_import.owl
INFO:root:  File: target/mondo-ingest/src/ontology/imports/omo_import.owl
INFO:root:  File: target/mondo-ingest/src/ontology/imports/ro_terms.txt
INFO:root:  File: target/mondo-ingest/src/ontology/imports/omo_terms.txt
INFO:root:  File: target/mondo-ingest/src/sparql/mondo-ingest_terms.sparql
INFO:root:  File: target/mondo-ingest/src/ontology/components/doid.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/gard.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/icd10cm.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/icd10who.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/ncit.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/omim.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/ordo.owl
INFO:root:  File: target/mondo-ingest/src/ontology/profile.txt
INFO:root:  File: target/mondo-ingest/.github/workflows/qc.yml
INFO:root:  File: target/mondo-ingest/docs/index.md
INFO:root:  File: target/mondo-ingest/docs/cite.md
INFO:root:  File: target/mondo-ingest/docs/contributing.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/index.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/ContinuousIntegration.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/EditorsWorkflow.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/ReleaseWorkflow.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/RepoManagement.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/RepositoryFileStructure.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/SettingUpDockerForODK.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/UpdateImports.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/components.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/ManageDocumentation.md
INFO:root:  File: target/mondo-ingest/mkdocs.yaml
INFO:root:  File: target/mondo-ingest/.github/workflows/docs.yml
INFO:root:  File: target/mondo-ingest/README.md
INFO:root:  File: target/mondo-ingest/CONTRIBUTING.md
INFO:root:  File: target/mondo-ingest/issue_template.md
INFO:root:  File: target/mondo-ingest/.gitignore
INFO:root:  File: target/mondo-ingest/src/scripts/validate_id_ranges.sc
INFO:root:  File: target/mondo-ingest/src/scripts/update_repo.sh
INFO:root:  File: target/mondo-ingest/src/sparql/postprocess-module.ru
INFO:root:  File: target/mondo-ingest/src/sparql/preprocess-module.ru
INFO:root:  File: target/mondo-ingest/src/sparql/edges.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/basic-report.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/obsoletes.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/synonyms.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/labels.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/xrefs.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/terms.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/subsets-labeled.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/simple-seed.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/README.md
INFO:root:  File: target/mondo-ingest/src/sparql/inject-subset-declaration.ru
INFO:root:  File: target/mondo-ingest/src/sparql/inject-synonymtype-declaration.ru
INFO:root:  File: target/mondo-ingest/src/ontology/run.bat
INFO:root:  File: target/mondo-ingest/src/ontology/README-editors.md
INFO:root:  File: target/mondo-ingest/src/ontology/Makefile
INFO:root:  File: target/mondo-ingest/src/ontology/catalog-v001.xml
INFO:root:  File: target/mondo-ingest/src/ontology/run.sh
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest-odk.yaml
Repository files have been successfully copied, but no git commands have been run.
total 0
drwxr-xr-x 14 odkuser dialout 448 Sep  4 19:37 ontology
drwxr-xr-x  4 odkuser dialout 128 Sep  4 19:37 scripts
drwxr-xr-x 17 odkuser dialout 544 Sep  4 19:37 sparql
total 0
drwxr-xr-x  4 odkuser dialout  128 Apr 13 22:00 analysis
drwxr-xr-x 12 odkuser dialout  384 Aug 30 22:35 mappings
drwxr-xr-x 27 odkuser dialout  864 Sep  4 19:37 ontology
drwxr-xr-x 21 odkuser dialout  672 Aug 30 22:35 scripts
drwxr-xr-x 72 odkuser dialout 2304 Aug 30 22:35 sparql
eepWARNING: These files should be manually migrated: mkdocs.yaml, .gitignore, src/ontology/catalog.xml (if you added a new import or component)
WARNING: Your QC workflows have not been updated automatically. Please update the ODK version number(s) in .github/workflows/qc.yml.
Ontology repository update successfully completed.
### DEBUG STATS ###
Elapsed time: 0:02.22
Peak memory: 35588 kb
Please remember to update your ODK image from time to time: https://oboacademy.github.io/obook/howto/odk-update/.



(mondo-ingest) git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   Makefile
	modified:   metadata/gard.yml

no changes added to commit (use "git add" and/or "git commit -a")



(mondo-ingest) sh run.sh make update_repo; beep
Running  with -Xmx20G of memory for ROBOT and Java-based pipeline steps.
mondo-ingest.Makefile:39: warning: overriding recipe for target 'components/omim.owl'
Makefile:447: warning: ignoring old recipe for target 'components/omim.owl'
mondo-ingest.Makefile:54: warning: overriding recipe for target 'components/ordo.owl'
Makefile:458: warning: ignoring old recipe for target 'components/ordo.owl'
mondo-ingest.Makefile:73: warning: overriding recipe for target 'components/ncit.owl'
Makefile:436: warning: ignoring old recipe for target 'components/ncit.owl'
mondo-ingest.Makefile:84: warning: overriding recipe for target 'components/doid.owl'
Makefile:392: warning: ignoring old recipe for target 'components/doid.owl'
mondo-ingest.Makefile:106: warning: overriding recipe for target 'component-download-icd10cm.owl'
Makefile:410: warning: ignoring old recipe for target 'component-download-icd10cm.owl'
mondo-ingest.Makefile:112: warning: overriding recipe for target 'components/icd10cm.owl'
Makefile:414: warning: ignoring old recipe for target 'components/icd10cm.owl'
mondo-ingest.Makefile:127: warning: overriding recipe for target 'components/icd10who.owl'
Makefile:425: warning: ignoring old recipe for target 'components/icd10who.owl'
mondo-ingest.Makefile:142: warning: overriding recipe for target 'component-download-gard.owl'
Makefile:399: warning: ignoring old recipe for target 'component-download-gard.owl'
mondo-ingest.Makefile:146: warning: overriding recipe for target 'components/gard.owl'
Makefile:403: warning: ignoring old recipe for target 'components/gard.owl'
mondo-ingest.Makefile:151: warning: overriding recipe for target 'mondo-ingest-full.owl'
Makefile:588: warning: ignoring old recipe for target 'mondo-ingest-full.owl'
mondo-ingest.Makefile:545: warning: overriding recipe for target 'help'
Makefile:659: warning: ignoring old recipe for target 'help'
sh ../scripts/update_repo.sh
This (experimental) update script will create a new repo according to your config file. It will:
(1) overwrite your repositories Makefile, ODK sparql queries (your custom queries wont be touched) and docker wrapper (run.sh).
(2) and add missing files, if any.
INFO:root:  Copying: /tools/templates/_dynamic_files.jinja2 -> target/mondo-ingest/_dynamic_files.jinja2
INFO:root:  Copying: /tools/templates/README.md.jinja2 -> target/mondo-ingest/README.md.jinja2
INFO:root:  Copying: /tools/templates/CODE_OF_CONDUCT.md -> target/mondo-ingest/CODE_OF_CONDUCT.md
INFO:root:  Copying: /tools/templates/CONTRIBUTING.md.jinja2 -> target/mondo-ingest/CONTRIBUTING.md.jinja2
INFO:root:  Copying: /tools/templates/issue_template.md.jinja2 -> target/mondo-ingest/issue_template.md.jinja2
INFO:root:  Copying: /tools/templates/.gitignore.jinja2 -> target/mondo-ingest/.gitignore.jinja2
INFO:root:Applying templates
INFO:root:  Unpacking: target/mondo-ingest/_dynamic_files
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/mondo-ingest-edit.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/mondo-ingest-idranges.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/mondo-ingest.Makefile
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/ro_import.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/omo_import.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/ro_terms.txt
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/imports/omo_terms.txt
INFO:root:  Unpacking into: target/mondo-ingest/src/sparql/mondo-ingest_terms.sparql
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/doid.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/gard.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/icd10cm.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/icd10who.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/ncit.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/omim.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/components/ordo.owl
INFO:root:  Unpacking into: target/mondo-ingest/src/ontology/profile.txt
INFO:root:  Unpacking into: target/mondo-ingest/.github/workflows/qc.yml
INFO:root:  Unpacking into: target/mondo-ingest/docs/index.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/cite.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/contributing.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/index.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/ContinuousIntegration.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/EditorsWorkflow.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/ReleaseWorkflow.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/RepoManagement.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/RepositoryFileStructure.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/SettingUpDockerForODK.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/UpdateImports.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/components.md
INFO:root:  Unpacking into: target/mondo-ingest/docs/odk-workflows/ManageDocumentation.md
INFO:root:  Unpacking into: target/mondo-ingest/mkdocs.yaml
INFO:root:  Unpacking into: target/mondo-ingest/.github/workflows/docs.yml
INFO:root:  Compiling: /tools/templates/README.md.jinja2 -> target/mondo-ingest/README.md
INFO:root:  Compiling: /tools/templates/CONTRIBUTING.md.jinja2 -> target/mondo-ingest/CONTRIBUTING.md
INFO:root:  Compiling: /tools/templates/issue_template.md.jinja2 -> target/mondo-ingest/issue_template.md
INFO:root:  Compiling: /tools/templates/.gitignore.jinja2 -> target/mondo-ingest/.gitignore
INFO:root:Applying templates
INFO:root:  Copying: /tools/templates/src/scripts/validate_id_ranges.sc -> target/mondo-ingest/src/scripts/validate_id_ranges.sc
INFO:root:  Copying: /tools/templates/src/scripts/update_repo.sh.jinja2 -> target/mondo-ingest/src/scripts/update_repo.sh.jinja2
INFO:root:Applying templates
INFO:root:  Compiling: /tools/templates/src/scripts/update_repo.sh.jinja2 -> target/mondo-ingest/src/scripts/update_repo.sh
INFO:root:  Copying: /tools/templates/src/sparql/postprocess-module.ru -> target/mondo-ingest/src/sparql/postprocess-module.ru
INFO:root:  Copying: /tools/templates/src/sparql/preprocess-module.ru -> target/mondo-ingest/src/sparql/preprocess-module.ru
INFO:root:  Copying: /tools/templates/src/sparql/README.md.jinja2 -> target/mondo-ingest/src/sparql/README.md.jinja2
INFO:root:  Copying: /tools/templates/src/sparql/edges.sparql -> target/mondo-ingest/src/sparql/edges.sparql
INFO:root:  Copying: /tools/templates/src/sparql/basic-report.sparql -> target/mondo-ingest/src/sparql/basic-report.sparql
INFO:root:  Copying: /tools/templates/src/sparql/obsoletes.sparql -> target/mondo-ingest/src/sparql/obsoletes.sparql
INFO:root:  Copying: /tools/templates/src/sparql/synonyms.sparql -> target/mondo-ingest/src/sparql/synonyms.sparql
INFO:root:  Copying: /tools/templates/src/sparql/inject-subset-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-subset-declaration.ru.jinja2
INFO:root:  Copying: /tools/templates/src/sparql/labels.sparql -> target/mondo-ingest/src/sparql/labels.sparql
INFO:root:  Copying: /tools/templates/src/sparql/xrefs.sparql -> target/mondo-ingest/src/sparql/xrefs.sparql
INFO:root:  Copying: /tools/templates/src/sparql/inject-synonymtype-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-synonymtype-declaration.ru.jinja2
INFO:root:  Copying: /tools/templates/src/sparql/terms.sparql -> target/mondo-ingest/src/sparql/terms.sparql
INFO:root:  Copying: /tools/templates/src/sparql/subsets-labeled.sparql -> target/mondo-ingest/src/sparql/subsets-labeled.sparql
INFO:root:  Copying: /tools/templates/src/sparql/simple-seed.sparql -> target/mondo-ingest/src/sparql/simple-seed.sparql
INFO:root:Applying templates
INFO:root:  Compiling: /tools/templates/src/sparql/README.md.jinja2 -> target/mondo-ingest/src/sparql/README.md
INFO:root:  Compiling: /tools/templates/src/sparql/inject-subset-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-subset-declaration.ru
INFO:root:  Compiling: /tools/templates/src/sparql/inject-synonymtype-declaration.ru.jinja2 -> target/mondo-ingest/src/sparql/inject-synonymtype-declaration.ru
INFO:root:  Copying: /tools/templates/src/ontology/run.bat.jinja2 -> target/mondo-ingest/src/ontology/run.bat.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/README-editors.md.jinja2 -> target/mondo-ingest/src/ontology/README-editors.md.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/Makefile.jinja2 -> target/mondo-ingest/src/ontology/Makefile.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/catalog-v001.xml.jinja2 -> target/mondo-ingest/src/ontology/catalog-v001.xml.jinja2
INFO:root:  Copying: /tools/templates/src/ontology/run.sh.jinja2 -> target/mondo-ingest/src/ontology/run.sh.jinja2
INFO:root:Applying templates
INFO:root:  Compiling: /tools/templates/src/ontology/run.bat.jinja2 -> target/mondo-ingest/src/ontology/run.bat
INFO:root:  Compiling: /tools/templates/src/ontology/README-editors.md.jinja2 -> target/mondo-ingest/src/ontology/README-editors.md
INFO:root:  Compiling: /tools/templates/src/ontology/Makefile.jinja2 -> target/mondo-ingest/src/ontology/Makefile
INFO:root:  Compiling: /tools/templates/src/ontology/catalog-v001.xml.jinja2 -> target/mondo-ingest/src/ontology/catalog-v001.xml
INFO:root:  Compiling: /tools/templates/src/ontology/run.sh.jinja2 -> target/mondo-ingest/src/ontology/run.sh
INFO:root:Created files:
INFO:root:  File: target/mondo-ingest/CODE_OF_CONDUCT.md
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest-edit.owl
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest-idranges.owl
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest.Makefile
INFO:root:  File: target/mondo-ingest/src/ontology/imports/ro_import.owl
INFO:root:  File: target/mondo-ingest/src/ontology/imports/omo_import.owl
INFO:root:  File: target/mondo-ingest/src/ontology/imports/ro_terms.txt
INFO:root:  File: target/mondo-ingest/src/ontology/imports/omo_terms.txt
INFO:root:  File: target/mondo-ingest/src/sparql/mondo-ingest_terms.sparql
INFO:root:  File: target/mondo-ingest/src/ontology/components/doid.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/gard.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/icd10cm.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/icd10who.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/ncit.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/omim.owl
INFO:root:  File: target/mondo-ingest/src/ontology/components/ordo.owl
INFO:root:  File: target/mondo-ingest/src/ontology/profile.txt
INFO:root:  File: target/mondo-ingest/.github/workflows/qc.yml
INFO:root:  File: target/mondo-ingest/docs/index.md
INFO:root:  File: target/mondo-ingest/docs/cite.md
INFO:root:  File: target/mondo-ingest/docs/contributing.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/index.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/ContinuousIntegration.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/EditorsWorkflow.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/ReleaseWorkflow.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/RepoManagement.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/RepositoryFileStructure.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/SettingUpDockerForODK.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/UpdateImports.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/components.md
INFO:root:  File: target/mondo-ingest/docs/odk-workflows/ManageDocumentation.md
INFO:root:  File: target/mondo-ingest/mkdocs.yaml
INFO:root:  File: target/mondo-ingest/.github/workflows/docs.yml
INFO:root:  File: target/mondo-ingest/README.md
INFO:root:  File: target/mondo-ingest/CONTRIBUTING.md
INFO:root:  File: target/mondo-ingest/issue_template.md
INFO:root:  File: target/mondo-ingest/.gitignore
INFO:root:  File: target/mondo-ingest/src/scripts/validate_id_ranges.sc
INFO:root:  File: target/mondo-ingest/src/scripts/update_repo.sh
INFO:root:  File: target/mondo-ingest/src/sparql/postprocess-module.ru
INFO:root:  File: target/mondo-ingest/src/sparql/preprocess-module.ru
INFO:root:  File: target/mondo-ingest/src/sparql/edges.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/basic-report.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/obsoletes.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/synonyms.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/labels.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/xrefs.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/terms.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/subsets-labeled.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/simple-seed.sparql
INFO:root:  File: target/mondo-ingest/src/sparql/README.md
INFO:root:  File: target/mondo-ingest/src/sparql/inject-subset-declaration.ru
INFO:root:  File: target/mondo-ingest/src/sparql/inject-synonymtype-declaration.ru
INFO:root:  File: target/mondo-ingest/src/ontology/run.bat
INFO:root:  File: target/mondo-ingest/src/ontology/README-editors.md
INFO:root:  File: target/mondo-ingest/src/ontology/Makefile
INFO:root:  File: target/mondo-ingest/src/ontology/catalog-v001.xml
INFO:root:  File: target/mondo-ingest/src/ontology/run.sh
INFO:root:  File: target/mondo-ingest/src/ontology/mondo-ingest-odk.yaml
Repository files have been successfully copied, but no git commands have been run.
total 0
drwxr-xr-x 14 odkuser dialout 448 Sep  4 19:37 ontology
drwxr-xr-x  4 odkuser dialout 128 Sep  4 19:37 scripts
drwxr-xr-x 17 odkuser dialout 544 Sep  4 19:37 sparql
total 0
drwxr-xr-x  4 odkuser dialout  128 Apr 13 22:00 analysis
drwxr-xr-x 12 odkuser dialout  384 Aug 30 22:35 mappings
drwxr-xr-x 27 odkuser dialout  864 Sep  4 19:37 ontology
drwxr-xr-x 21 odkuser dialout  672 Aug 30 22:35 scripts
drwxr-xr-x 72 odkuser dialout 2304 Aug 30 22:35 sparql
WARNING: These files should be manually migrated: mkdocs.yaml, .gitignore, src/ontology/catalog.xml (if you added a new import or component)
WARNING: Your QC workflows have not been updated automatically. Please update the ODK version number(s) in .github/workflows/qc.yml.
Ontology repository update successfully completed.
### DEBUG STATS ###
Elapsed time: 0:01.79
Peak memory: 35676 kb
Please remember to update your ODK image from time to time: https://oboacademy.github.io/obook/howto/odk-update/.

(mondo-ingest) git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   Makefile
	modified:   metadata/gard.yml

no changes added to commit (use "git add" and/or "git commit -a")
```

</p>
</details>

Reply:
Is it somehow possible like we do in [OBO](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/mondo.md?plain=1#L88) to have the bottom part of the docs custom and the top part autogenerated (see make documentation goal I made) 

(copy of comment above)

Reply:
`make documentation`. That's what I needed! I thought this was done by `update_repo.

I suppose I will try to solve this problem of static vs dynamic docs in this PR (unless you want me to split it into a new PR). My goal would be that when the command below ultimately gets run, it doesn't completely overwrite the static part.
```make
../../docs/sources/%.md: metadata/%.yml | ../../docs/sources/
	j2 config/source_documentation.md.j2 $< > $@
```